src/App.css

```diff
@@ -2,4 +2,5 @@
 
 html {
   background-color: var(--bg-black);
+  accent-color:var(--prim-color);
```

Comment:
try to use full name for better understanding

Reply:
Changed all color variable to --color-colorname